src/main/resources/assets/structurize/gui/windowscantool.xml

```diff
@@ -36,8 +36,9 @@
 
     <list id="entities" size="200 100" pos="220 100">
```

Comment:
Looks ray somehow manage to port something wrong? or was it ported at all?

```suggestion
    <list id="entities" size="200 100" pos="220 100" childspacing="-1">
        <box size="100% 30" linewidth="1">
            <itemicon id="resourceIcon" size="16 16" pos="1 1"/>
            <label id="resourceName" size="172 12" pos="19 1" color="white" wrap="true"/>
            <buttonimage label="$(com.ldtteam.structurize.gui.scantool.remove)" id="removeEntity" pos="19 13" size="86 17"
                         source="structurize:textures/gui/builderhut/builder_button_medium.png" textcolor="black"/>
        </box>
    </list>
```

Reply:
can u just for sanity run git diff over all gui xmls? between 1.17 and 1.18
thanks a lot

Reply:
1.16 and 1.17 blockout differs so much, the xmls are very different in many regards (i.e. especially borders)

Reply:
fyi, i did not mention 1.16

Reply:
Ah, between 1.17 and 1.18 only, nvm

Reply:
Ok, my github lookup for that PR didn't show anything suspicious, just this list not being ported for some reason. So just fix this accordingly to my suggestion and we can merge.

Reply:
Done.